routes/user.js

```diff
@@ -0,0 +1,77 @@
+var express = require('express');
+var router = express.Router();
+var User = require('../models/index.js').User;
+var mail = require('../config/nodeMailer');
+
+router.get('/:username', isLoggedIn, function(req, res, next) {
+  var user = req.user;
+  res.render('profile', { title: user.username + "'s Profile", user: user});
```

Comment:
I was expecting this to return a JSON response. Is it HTML currently for dev/viewing purposes?